examples/lightclient/CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ include(${CMAKE_CURRENT_LIST_DIR}/../../core/wakaama.cmake)
 include(${CMAKE_CURRENT_LIST_DIR}/../shared/shared.cmake)
 
 add_definitions(-DLWM2M_CLIENT_MODE)
+if(LWM2M_VERSION VERSION_GREATER "1.0")
+    add_definitions(-DLWM2M_SUPPORT_SENML_JSON)
+else()
+    add_definitions(-DLWM2M_SUPPORT_JSON)
```

Comment:
TLV will be supported by default for LWM2M 1.0. For a lightweight client I don't think we need to add JSON also.